stack.yaml

```diff
@@ -1,14 +1,9 @@
-resolver: lts-4.2
+flags: {}
+nix:
+  enable: false
+  shell-file: stack-shell.nix
 packages:
 - '.'
-flags:
-   idris:
-     FFI: true
-     GMP: True
-extra-deps: 
-- annotated-wl-pprint-0.7.0
-- cheapskate-0.1.0.4
-- libffi-0.1
-nix:
-   enable: false
-   shell-file: stack-shell.nix
+extra-deps:
+- text-1.2.2.0
```

Comment:
This is already in LTS 5.5 afaik?
https://www.stackage.org/lts-5.5/package/text-1.2.2.0